### ISSUE TITLE
What is bad web design?

### ISSUE THREAD

**Issue by discordfonts123** [mozilla-mobile/fenix#24676]
A cluttered layout, hidden navigation menu, lack of color contrast, non-responsive design, and inconsistent typefaces are just a few characteristics that might make a website design bad. The defining characteristic of a bad website design is the lack of user-centricity.
